### DISCUSSION TITLE
hide sensitive details from log

### DISCUSSION THREAD

**Discussion by mazenaissa** [keycloak/keycloak#12751] [Archive]
Hello,
Prior to the keycloak quakus version distribution, we used to hide the ipadress & username from logs using this startup-script : 
```
embed-server --server-config=standalone-ha.xml --std-out=echo 
batch
/subsystem=logging/console-handler=CONSOLE:write-attribute(name=filter-spec, value="all(substituteAll(\"(?<=ipAddress=|username=)(.*?)(?=[,\\n]|$)\",\"******\"))")
run-batch 
stop-embedded-server
```
The startup scripts are no longer supported, any idea how can I achieve it on Keycloak 18?
Thanks,

**Comment by SeJunB** [comment]
Not sure if you still need this but I was able to achieve this by implementing a Quarkus extension. See https://github.com/SeJunB/KeycloakLogFilter for an example.
